versions.yaml

```diff
@@ -121,13 +121,13 @@ assets:
     architecture:
       aarch64:
         name: "ubuntu"
-        version: "latest"
+        version: "focal"
```

Comment:
Should update to `jammy`, the latest LTS version, which equals `latest`.

Reply:
Updated to jammy.